### PR TITLE
Py-radix: init at v0.10.0

### PR DESCRIPTION
**Pull request by mkgvt**
###### Motivation for this change

Packaged py-radix library for inclusion in nixpkgs.

###### Things done

Created pkgs/development/python-modules/py-radix/default.nix.
Modified pkgs/top-level/python-packages.nix.
Built against: Python 2.7, 3.5, 3.6, 3.7
Tested user installation: nix-env -i python36.pkgs.py-radix

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Assured whether relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Review by aanderse — COMMENTED** [review]
Did you mean to remove the `rename` package here?

**Comment by mkgvt** [comment]
On Sat, Mar 23, 2019 at 7:39 AM aanderse <notifications@github.com> wrote:

> *@aanderse* commented on this pull request.
>
> Did you mean to remove the rename package here?
>

Sorry. It seems I accidentally conflated multiple pull requests. I am a
GitHub novice. An earlier pull request was for the removal of rename.

Mark
-- 
Mark Gardner
--

**Comment by aanderse** [comment]
I figured as much, no problem. Are you familiar at all with the process of rewriting your git commits to cleanup these PRs?

**Comment by mkgvt** [comment]
On Sat, Mar 23, 2019 at 11:08 AM aanderse <notifications@github.com> wrote:

> I figured as much, no problem. Are you familiar at all with the process of
> rewriting your git commits to cleanup these PRs?
>
I have done it in git but not in GitHub.

Mark
-- 
Mark Gardner
--

**Comment by aanderse** [comment]
ping @mkgvt

**Comment by mkgvt** [comment]
Still here.

**Comment by aanderse** [comment]
Can you please cleanup the git history? I'm looking for a single commit. You also still need to remove the commit where you dropped a perl package.

File: pkgs/development/python-modules/py-radix/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  version = "v0.10.0";
+
+  src = fetchFromGitHub {
+    owner = "mjschultz";
+    repo = "py-radix";
+    rev = version;
+    sha256 = "01xyn9lg6laavnzczf5bck1l1c2718ihxx0hvdkclnnxjqhbrqis";
+  };
+
+  doCheck = true;
```
Comment (FRidh) [inline pkgs/development/python-modules/py-radix/default.nix]: this is the default, so remove

**Comment by mkgvt** [comment]
Hopefully I have fixed the issues with this PR in #61251.

**Pull request closed without merging**